GNUmakefile

```diff
@@ -0,0 +1,14 @@
+include $(GNUSTEP_MAKEFILES)/common.make
+
+BUNDLE_NAME = MIPSCPU
+MIPSCPU_OBJC_FILES = \
+	MIPSCPU/MIPSCPU.m \
+	MIPSCPU/MIPSCtx.m
+
+#NOWARN=-Wno-format -Wno-return-type -Wno-unused-value -Wno-unused-variable -Wno-self-assign
+MIPSCPU_OBJCFLAGS=$(NOWARN) \
+	-I${HOME}/GNUstep/Library/ApplicationSupport/Hopper/HopperSDK/include \
```

Comment:
Thank your for your PR.
I think it's preferable here to use the local HopperSDK, e.g.
```
    -I./HopperSDK/include \
```